data/index.html

```diff
@@ -5,133 +5,125 @@
 <html>
 <head>
 
-<meta charset="utf-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge">
-<meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-<title>Firefox data explorer - Find probes</title>
-<meta charset="utf-8" />
-<link rel="stylesheet" href="explorer.css">
-<style type="text/css">
-/*
-  table {
-    border-collapse: collapse;
-  }
-  table, th, td {
-    border: 1px solid black;
-  }
-  #notice {
-    font-style: italic;
-    margin: 5px;
-    padding: 5px 10px;
-    background-color: #EEEEEE;
-  }
-  #stats {
-    font-style: italic;
-    font-size: small;
-    margin-bottom: 20px;
-  }
-  #measurements h4 {
-    margin: 10px 5px 3px 0px;
-  }
-*/
-</style>
+  <title>Firefox data explorer - Find probes</title>
 
-<script type="text/javascript"
-        src="jquery-3.1.1.min.js">
-</script>
-<script type="text/javascript" src="explore.js"></script>
-
-<!-- Latest compiled and minified CSS -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-
-<!-- Optional theme -->
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
-
-<!-- Latest compiled and minified JavaScript -->
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+  <!-- CSS  -->
+  <!-- CSS:Framework BS 4 alpha -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+  <!-- CSS:Framework:Theme United -->
```

Comment:
Should this be Cosmo?

Reply:
Updated file name and link reference for clarity.